frontend/components/ActivityDetails/InstallDetails/AppInstallDetails/AppInstallDetails.tsx

```diff
@@ -74,8 +76,27 @@ export const AppInstallDetails = ({
 
   if (isLoading) {
     return <Spinner />;
-  } else if (isError) {
-    return <DataError description="Close this modal and try again." />;
+  }
+
+  if (isError) {
```

Comment:
small nit: you could check for `isError && error` so you don't have to have use the optional chaining operator when accessing `status` attribute